be/src/exec/vectorized/iceberg_delete_builder.h

```diff
@@ -0,0 +1,57 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#pragma once
+
+#include "common/status.h"
+#include "exec/vectorized/hdfs_scanner.h"
+
+namespace starrocks::vectorized {
+    class PositionDeleteBuilder {
+    public:
+        PositionDeleteBuilder() = default;
+        virtual ~PositionDeleteBuilder() = default;
+
+        virtual Status build(RuntimeState* runtime_state, std::unordered_set<std::int64_t> need_skip_rowids,
```

Comment:
do we really need to understand `RuntimeState`. can we just pass timezone

Reply:
and can we just pass `File` instead of `TIcebergDeleteFile` for more versability

Reply:
> do we really need to understand `RuntimeState`. can we just pass timezone

use RuntimeState can be more general